opentreemap/treemap/js/src/CanopyFilterControl.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+var $ = require('jquery'),
+    _ = require('lodash'),
+    L = require('leaflet'),
+    Bacon = require('baconjs');
+
+require('ionRangeSlider');
+
+var canopyFilterTmpl = _.template($('#canopy-filter-tmpl').html());
+
+var CanopyFilterControl = L.Control.extend({
+    options: {
+        position: 'bottomleft'
+    },
+
+    initialize: function() {
+        this._changeBus = new Bacon.Bus();
+        this.changeStream = this._changeBus.toEventStream();
+    },
+
+    onAdd: function(map) {
+        var self = this,
+            $el = $(canopyFilterTmpl());
+
+        $el.find('[name="canopy-slider"]').ionRangeSlider({
+            type: 'double',
+            min: 0,
+            max: 1,
+            from: 0,
+            to: 1,
+            step: 0.01,
+            drag_interval: true,
+            hide_min_max: true,
+            grid: false,
+            prettify: function(n) {
+                return Math.floor(n * 100) + '%';
```

Comment:
The slider jumps over certain values, but the values that are not selectable are not consistent. This may be frustrating for a user who is trying to set the sliders to "20% - 40%" but can only see "19% - 41%," for instance.

We may need to reverse the slider behavior so that the min and max are 0 to 100 and then divide by 100 in `onChange`.